packages/core-ethereum/src/index.spec.ts

```diff
@@ -22,3 +23,40 @@ describe(`HoprEthereum`, function () {
     log('successfully created the HoprEthereum instance.')
   })
 })
+
+describe('test HoprEthereum', function () {
+  const connector = new HoprCoreEthereum(dbMock, PARTY_A, stringToU8a(ACCOUNT_A.privateKey), sampleChainOptions)
+
+  it('should test isWhitelisted', async function () {
+    // @ts-ignore
+    connector.db = sinon.stub()
```

Comment:
Why don't use `HoprDB.createMock()` which creates an in-memory db? So probably no need to mock the methods individually